src/Microsoft.Azure.WebJobs.Logging/Entities/InstanceTableEntity.cs

```diff
@@ -90,9 +89,14 @@ internal static string RowKeyFunctionInstanceId(Guid functionInstanceId)
 
         public string TriggerReason { get; set; }
 
-        public DateTimeOffset StartTime { get; set; }
+        public DateTime StartTime { get; set; }
```

Comment:
Are these breaking format changes in the serialization format to table storage? I.e. existing Dashboard can read both old an new?